activesupport/test/log_subscriber_test.rb

```diff
@@ -80,10 +80,15 @@ def test_event_attributes
     instrument "some_event.my_log_subscriber"
     wait
     event = @log_subscriber.event
+    if defined?(JRUBY_VERSION)
+      assert_operator event.cpu_time, :>=, 0
+      assert_operator event.allocations, :>=, 0
```

Comment:
`cpu_time` and `allocations` always return 0, so `assert_equal 0, ...` is preferable.

Reply:
Thanks for the review and useful information. Updated and squashed.